pkg/defaults/complianceoperator/files/cis-benchmarks.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Do we have a follow on 4.5 ticket to add the rest of the mappings from https://docs.openshift.com/container-platform/4.15/security/compliance_operator/co-scans/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles?

Reply:
I'll open a ticket.